test/e2e/kubevirt.go

```diff
@@ -534,6 +542,22 @@ iperf3 -t 0 -c %[1]s -p %[2]d --logfile %[3]s &
 				Expect(err).NotTo(HaveOccurred())
 				return vmi.Status.MigrationState.Completed
 			}).WithPolling(time.Second).WithTimeout(20*time.Minute).Should(BeTrue(), "should complete migration")
+			Eventually(func() bool {
+				err := crClient.Get(context.TODO(), crclient.ObjectKeyFromObject(vmi), vmi)
+				Expect(err).NotTo(HaveOccurred())
+				if vmi.Status.MigrationState.SourcePod == "" {
+					return false
+				}
+				sourcePod := &corev1.Pod{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: namespace,
+						Name:      vmi.Status.MigrationState.SourcePod,
+					},
+				}
+				err = crClient.Get(context.TODO(), crclient.ObjectKeyFromObject(sourcePod), sourcePod)
+				Expect(err).NotTo(HaveOccurred())
+				return sourcePod.Status.Phase == corev1.PodSucceeded
+			}).WithPolling(time.Second).WithTimeout(time.Minute).Should(BeTrue(), "should move source pod to Completed")
```

Comment:
Since we  should not check SourcePod at eventually (it should be there already if not is an error) we can directly return Status.Phase at the Eventually function
```golang
func()  corev1.PodPhase
...
return sourcePod.Status.Phase
}).WithPolling(time.Second).WithTimeout(time.Minute).Should(corev1.PodSucceeded(), "should move source pod to Completed")

```

Reply:
DONE